theme/templates/pages/group.html

```diff
@@ -89,23 +89,33 @@ <h4 class="text-muted">{{ group.gaccess.purpose|linebreaks }}</h4>
         <div class="full-width-tabs-container" id="user-profile-tabs">
             <!-- Nav tabs -->
             <ul class="nav nav-tabs full-width-tabs" role="tablist">
-                {% if profile_user.is_group_owner or profile_user.is_group_editor or profile_user.is_group_viewer %}
+{#                {% if profile_user.is_group_owner or profile_user.is_group_editor or profile_user.is_group_viewer %}#}
```

Comment:
Remove this commented out code and empty lines below if not needed anymore.

Reply:
Fixed in 4c23d114c